src/crimson/osd/ops_executer.cc

```diff
@@ -79,14 +79,14 @@ OpsExecuter::interruptible_call_errorator::future<> OpsExecuter::do_op_call(OSDO
                  cname, mname, num_read, num_write);
   const auto prev_rd = num_read;
   const auto prev_wr = num_write;
-  return seastar::async(
+  return interruptor::async(
```

Comment:
There is another call to `seastar::async` (in `main()`) but I doubt we need to switch it.